File: jesusp/Rectangulo.c
#include <stdio.h>

main()
{
 int lado1;
 int lado2;
 int perimetro;
 int area;
 lado1=0;
 lado2=0;
 perimetro=0;
 area=0;
 printf("Los lados verticales:");
 scanf("%d", &lado1);
 printf("Los lados horizontales:");
 scanf("%d", &lado2);
 perimetro=(lado1+lado1)+(lado2+lado2);
 area=lado1*lado2;
 printf("El perimetro es:");
 printf("%d", perimetro);
 printf("\n El area es:");
 printf("%d", area);
 
}

File: jesusp/NetBeansProjects/perimetr/src/main/java/com/mycompany/perimetr/rectangulo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.perimetr;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.logging.Level;
import java.util.logging.Logger;

/**
 *
 * @author Usuario
 */
public class rectangulo {
     public static void main(String[] args)
{
         try {
             int lado1;
             int lado2;
             int perimetro;
             int area;          
             
             System.out.println("Los lados verticales:");
             BufferedReader in =
                     new BufferedReader(new InputStreamReader(System.in));             
             lado1 = Integer.parseInt(in.readLine());
             lado2 = Integer.parseInt(in.readLine());
             
             perimetro=(lado1+lado1)+(lado2+lado2);
             area=lado1*lado2;
             
             System.out.printf("El perimetro es:");
             
             System.out.printf("%d", perimetro);
             
             System.out.printf("\n El area es:");
             
             System.out.printf("%d", area);
         } catch (IOException ex) {
             System.out.println("Ha habido un error");
         }
 
}
    
}
